panda/src/cocoadisplay/cocoaGraphicsWindow.mm

```diff
@@ -1031,10 +1056,31 @@
       properties.set_cursor_filename(cursor_filename);
       properties.clear_cursor_filename();
     } else {
-      NSImage *image = load_image(cursor_filename);
+      NSData *image_data = load_image_data(cursor_filename);
+
+      // Read the metadata from the image, which should contain hotspotX and
+      // hotspotY properties.
+      CGImageSourceRef cg_image = CGImageSourceCreateWithData((CFDataRef)image_data, nullptr);
+      NSDictionary *image_props = (NSDictionary *)CGImageSourceCopyPropertiesAtIndex(cg_image, 0, nil);
```

Comment:
This code is duplicated, perhaps we could create a `load_cursor` method that returned an NSCursor pointer?  Or, failing that, a load_image_data that took the CGFloat hotspot_x/y as reference arguments?